dist/index.d.ts

```diff
@@ -90,20 +94,20 @@ export interface FieldConfigExtension<TSource, TContext, TArgs> {
 
 export interface UnionTypeExtension {
   sqlTable?: ThunkWithArgsCtx<string, any, any>
-  uniqueKey?: string | string[]
-  alwaysFetch?: string | string[]
+  uniqueKey?: ArrayOrValue<string>
+  alwaysFetch?: ArrayOrValue<ThunkWithArgsCtx<string, any, any>>
```

Comment:
I think this should be a Thunk<ArrayOrValue as opposed to an ArrayOrValue<Thunk< right? One thunk that can return many thingies?